<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>简单网页浏览器</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar_url">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_B"/>
   <addaction name="action_F"/>
   <addaction name="action_R"/>
   <addaction name="action_Stop"/>
   <addaction name="action_HOME"/>
  </widget>
  <widget class="QToolBar" name="toolBar_bookmark">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="separator"/>
    <addaction name="action_CW"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>导航</string>
    </property>
    <addaction name="action_F"/>
    <addaction name="action_B"/>
    <addaction name="action_R"/>
    <addaction name="action_Stop"/>
    <addaction name="separator"/>
    <addaction name="action_HOME"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>收藏</string>
    </property>
    <addaction name="action_sc2bar"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>窗体</string>
    </property>
    <addaction name="action_I"/>
    <addaction name="action_O"/>
    <addaction name="action_Reset"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <action name="action_New">
   <property name="text">
    <string>新建页面(&amp;N)</string>
   </property>
   <property name="statusTip">
    <string>新建浏览页面</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>退出浏览器(&amp;Q)</string>
   </property>
   <property name="statusTip">
    <string>退出浏览器</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_CW">
   <property name="text">
    <string>关闭当前页面(&amp;W)</string>
   </property>
   <property name="statusTip">
    <string>关闭当前页面</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_H">
   <property name="text">
    <string>浏览记录(&amp;H)</string>
   </property>
   <property name="statusTip">
    <string>调出浏览记录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_F">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/res/forward.ico</normaloff>:/image/res/forward.ico</iconset>
   </property>
   <property name="text">
    <string>前进(&amp;F)</string>
   </property>
   <property name="statusTip">
    <string>前进页面</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="action_B">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/res/back.ico</normaloff>:/image/res/back.ico</iconset>
   </property>
   <property name="text">
    <string>后退(&amp;B)</string>
   </property>
   <property name="statusTip">
    <string>后退页面</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="action_R">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/res/reload.ico</normaloff>:/image/res/reload.ico</iconset>
   </property>
   <property name="text">
    <string>刷新(&amp;R)</string>
   </property>
   <property name="statusTip">
    <string>刷新页面</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="statusTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>剪贴(&amp;X)</string>
   </property>
   <property name="statusTip">
    <string>剪贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_P">
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
   <property name="statusTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Z">
   <property name="text">
    <string>撤销(&amp;Z)</string>
   </property>
   <property name="statusTip">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Y">
   <property name="text">
    <string>重做(&amp;Y)</string>
   </property>
   <property name="statusTip">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
   <property name="statusTip">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
   <property name="statusTip">
    <string>查找内容</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_sc">
   <property name="text">
    <string>添加收藏</string>
   </property>
   <property name="statusTip">
    <string>将网址收藏</string>
   </property>
  </action>
  <action name="action_sc2bar">
   <property name="text">
    <string>添加收藏到工具栏</string>
   </property>
   <property name="statusTip">
    <string>将网址收藏到工具栏</string>
   </property>
  </action>
  <action name="action_showsc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示收藏</string>
   </property>
   <property name="statusTip">
    <string>显示收藏</string>
   </property>
  </action>
  <action name="action_I">
   <property name="text">
    <string>放大(&amp;I)</string>
   </property>
   <property name="statusTip">
    <string>放大</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>缩小(&amp;O)</string>
   </property>
   <property name="statusTip">
    <string>缩小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_Reset">
   <property name="text">
    <string>恢复(&amp;R)</string>
   </property>
   <property name="statusTip">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/res/stop.ico</normaloff>:/image/res/stop.ico</iconset>
   </property>
   <property name="text">
    <string>停止(&amp;S)</string>
   </property>
   <property name="statusTip">
    <string>停止载入</string>
   </property>
  </action>
  <action name="action_HOME">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/res/home.png</normaloff>:/image/res/home.png</iconset>
   </property>
   <property name="text">
    <string>主页(&amp;H)</string>
   </property>
   <property name="statusTip">
    <string>载入主页</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
